packages/@react-aria/datepicker/docs/useDateRangePicker.mdx

```diff
@@ -543,7 +543,7 @@ function Example() {
   return (
     <>
       <DateRangePicker label="Date range" value={range} onChange={setRange} />
-      <p>Selected date: {formatter.formatRange(range.start.toDate(getLocalTimeZone()), range.end.toDate(getLocalTimeZone()))}</p>
+      <p>Selected date: {range ? formatter.formatRange(range.start.toDate(getLocalTimeZone()), range.end.toDate(getLocalTimeZone())) : '--'}</p>
```

Comment:
Do we need the '--' for when the value is null? I think it would be okay if it were just left blank (and this goes for the other ones too). I guess it depends if we want to indicate visually that there is no value besides just leaving it blank but open to opinions

Reply:
We were doing the '--' in RSP, so I replicated that to the other examples. I liked that the '--' is like an empty segment, but non-date component examples are more likely to have nothing.

Reply:
Ahh I see...thanks for letting me know. In that case, I guess we should just leave it as '--' like you had